src/data/finset/basic.lean

```diff
@@ -520,6 +520,9 @@ by rw [←coe_ssubset, coe_singleton, set.ssubset_singleton_iff, coe_eq_empty]
 lemma eq_empty_of_ssubset_singleton {s : finset α} {x : α} (hs : s ⊂ {x}) : s = ∅ :=
 ssubset_singleton_iff.1 hs
 
+lemma eq_singleton_or_nontrivial (ha : a ∈ s) : s = {a} ∨ (s : set α).nontrivial :=
```

Comment:
For consistency with the rest of the section:
```suggestion
lemma eq_singleton_or_nontrivial {s : finset α} {a : α} (ha : a ∈ s) : s = {a} ∨ (s : set α).nontrivial :=
```

Reply:
Well, depends on what bit of the section we're talking about.